service.subtitles.torec/resources/lib/TorecSubtitlesDownloader.py

```diff
@@ -198,18 +181,41 @@ def search_movie(self, movie_name):
 
         return self._fetch_subtitles_options(main_url)
 
-    def get_download_link(self, sub_id, option_id):
-        params    = {
-            "sub_id":     sub_id,
-            "code":       option_id,
-            "sh":         "yes",
-            "guest":      self._request_subtitle(sub_id),
-            "timewaited": 9
+    def _request_subtitle(self, sub_id):
+        params = {
+            "sub_id"  : sub_id, 
+            "s"       : 1440
         }
 
-        response = self.opener.open("%s/ajax/sub/downloadun.asp" % self.BASE_URL, urllib.urlencode(params))
+        response = self.opener.open("%s/ajax/sub/guest_time.asp" % self.BASE_URL, urllib.urlencode(params))
         return response.read()
 
+    def get_download_link(self, sub_id, option_id):
+        guest_token = self._request_subtitle(sub_id)
+
+        download_link = None
+        for i in xrange(1, 15):
```

Comment:
true, the website limit can go up to 13, so wanted to be on the safe side.
added usage of xbmc monitor